src/sequential/encryptor.rs

```diff
@@ -112,13 +124,16 @@ impl<S> Debug for State<S> {
 /// Due to the reduced complexity, a side effect is that this encryptor outperforms `SelfEncryptor`,
 /// particularly for small data (below `MIN_CHUNK_SIZE * 3` bytes) where no chunks are generated.
 // #[async_trait]
-pub struct Encryptor<S> {
-    state: Rc<RefCell<State<S>>>,
+pub struct Encryptor<S: Storage + 'static + Send + Sync>
+// where
+//     S: Storage + 'static + Send + Sync
```

Comment:
Just a minor thing, un-comment this out to make it consistent with other functions definitions.